lib/managers/files.js

```diff
@@ -1359,10 +1359,20 @@ Files.prototype.representation = {
  * @param {string} fileID - Box ID of the file being requested
  * @param {client.files.representation} representationType - The x-rep-hints value the application should create a
  *    representation for. This value can either come from FileRepresentationType enum or manually created
+ * @param {Object} [options] - Additional options. Can be left empty
+ * @param {Boolean} [options.fullInfo] - Set to true to return representation info where all states resolve to success. Default is false
  * @param {Function} [callback] - Passed an array of representaton objects if successful
  * @returns {Promise<Object>} A promise resolving to the representation response objects
 */
-Files.prototype.getRepresentationInfo = function(fileID, representationType, callback) {
+
+Files.prototype.getRepresentationInfo = function(fileID, representationType, options, callback) {
+	if (!representationType) {
```

Comment:
This check is likely not backwards-compatible, since this method would previously worked without specifying X-Rep-Hints.  You'll probably need to add checks later when handling the response to ensure that your code works correctly when no X-Rep-Hints are passed